src/foraging_gui/Foraging.py

```diff
@@ -2049,50 +2121,76 @@ def _Open_getListOfMice(self):
                     break
         return mice  
 
-    def _Open(self,open_last = False):
-
-        # stop current session first
-        self._StopCurrentSession() 
-
-        # Start new session
-        new_session = self._NewSession()
-        if not new_session:
-            return
-
-        if open_last:
-            mice = self._Open_getListOfMice()
-            W = MouseSelectorDialog(self, mice)
+    def _Open(self,open_last = False,input_file = ''):
+        if input_file == '' or input_file == False:
```

Comment:
What is input_file? What is it sometimes a string, and sometimes a boolean?

Reply:
It's a history issue. I deleted open_last = False